packages/kit/test/apps/hash-based-routing/src/routes/b/[slug]/+page.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+	import { page } from '$app/state';
+	let { data } = $props();
+</script>
+
+<p data-data>
+	{JSON.stringify(data.params)}
+	{data.route.id}
+	<!-- can't access url.hash from load function -->
```

Comment:
this restriction exists because we didn't want a discrepancy between server and client invocations of universal `load` functions, but if SSR is disabled in hash mode then maybe it makes sense to allow it?

Reply:
Yeah was thinking the same but didn't want to get side tracked